test/internals/reply.test.js

```diff
@@ -46,6 +46,16 @@ test('reply.serializer should set a custom serializer', t => {
   t.equal(reply._serializer, 'serializer')
 })
 
+test('reply.serialize should serialize payload', t => {
+  t.plan(1)
+  const reply = new Reply(null, null, null)
```

Comment:
you will need to pass a res to Reply, we read the status cose from there.

Reply:
<img width="452" alt="screen shot 2018-03-21 at 11 18 19 am" src="https://user-images.githubusercontent.com/16144158/37689530-9cf5b480-2cf9-11e8-81e6-1023d1b30ece.png">

Reply:
It's returning JSON object because responseSchema is empty. Figuring out what I need to put there (ref validation.js line 85-87)

Reply:
It looks like this method is meant to default to normal JSON serializer. Modifying test now.